readme.md

```diff
@@ -2,99 +2,115 @@
 
-Today is all about exploring some of the [new features](https://github.com/lukehoban/es6features) and getting comfortable with
-the new syntax.
+Today is all about exploring some of the [new features](https://github.com/lukehoban/es6features) and getting comfortable with the new syntax.
 
-> For more backstory, we recommend checking out [this talk](https://www.youtube.com/watch?v=PlmsweSNhTw) from Brendan Eich on what he views as the future of JS.
+> For more backstory, we recommend checking out [this talk](https://www.youtube.com/watch?v=PlmsweSNhTw) from Brendan Eich on what he views as the future of Javascript.
 
 ## New Features
 
-### Block Scope (10 / 25)
+### Block Scope (10 minutes / 0:15)
 
 <details>
-<summary>What does the concept of scope refer to in JS?</summary>
+  <summary><strong>What does the concept of scope refer to in Javascript?</strong></summary>
 
-In short, the notion of which variables are available where.
+  > In short, the notion of which variables are available where.
 
 </details>
 
----
+<br>
 
 <details>
-<summary>So far in class, what is the primary way to control scope in JS? Why do we want to control scope?</summary>
-
-Functions create new local scopes.
+  <summary><strong>What is the primary way to control scope in Javascript? Why do we want to control scope?</strong></summary>
 
-Scope allows us to use generic variable names (eg `count`, `input`, `evt`) while considering them in an isolated space.
-This makes our code easier to reason about.
+  > Functions create new local scopes.
+  >
+  > Scope allows us to use generic variable names using `var` while considering them in an isolated space. This makes our code easier to reason about.
 
 </details>
 
+<br>
+
+Let's revisit a code snippet from your "Scope & Context" lesson...
+
+```js
+teamName = "Giraffes" // What scope is this?
+var teamCity = "Sioux Falls" // What scope is this?
+
+function playBaseball() {
+  console.log("From " + teamCity + "...") // Does this work?
+  console.log("Welcome the " + teamName + "!") // Does this work?
+
+  pitcherName    = "Jesse Shawl" // What scope is this?
+  var batterName = "Breece Horper" // What scope is this?
+
+  console.log(batterName)  // Does this work?
+  console.log(pitcherName) // Does this work?
+}
+```
+
 #### `let`
 
-The fact that blocks of other special forms (`if`, `for`, `while`, etc) *do not* create scopes is one of the oddities for which JS does and should get knocked.
+The fact that blocks of other special forms (`if`, `for`, `while`, etc) *do not* create scopes is one of the oddities for which Javascript sometimes gets knocked. Other languages -- like Python, C++ and Java -- have had this feature for a long time.
+
```

Comment:
What about saying ES5 Javascript? This opening sentence might suggest to some students that JS as a whole doesn't create scopes for any set of {}